recipe/meta.yaml

```
@@ -150,8 +150,10 @@ outputs:
       string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
       detect_binary_files_with_prefix: False
       # weigh down cpu implementation and give cuda preference
-      track_features:
-        - pytorch-cpu                                      # [cuda_compiler_version == "None"]
+      track_features:                                           # [cuda_compiler_version == "None"]
+        - pytorch-cpu                                           # [cuda_compiler_version == "None"]
+      run_exports:                                              # [cuda_compiler_version == "None"]
```

```suggestion
      run_exports:                                              # [cuda_compiler_version == "None"]
```

I'm not sure about this syntax...